### ISSUE TITLE
Maybe add a removeAllCookies method :)

### ISSUE THREAD

**Issue by sbrighiu** [MaxToyberman/react-native-ssl-pinning#114]
TL;DR I'm lazy. Wanted to call removeAllCookies(), but settled for removeCookieByName('*');

diff --git a/node_modules/react-native-ssl-pinning/android/src/main/java/com/toyberman/RNSslPinningModule.java b/node_modules/react-native-ssl-pinning/android/src/main/java/com/toyberman/RNSslPinningModule.java
index 6fb4c84..9043dc5 100644
--- a/node_modules/react-native-ssl-pinning/android/src/main/java/com/toyberman/RNSslPinningModule.java
+++ b/node_modules/react-native-ssl-pinning/android/src/main/java/com/toyberman/RNSslPinningModule.java
@@ -153,9 +153,11 @@ public class RNSslPinningModule extends ReactContextBaseJavaModule {
 
             cookies = cookieStore.get(domain);
             if (cookies != null) {
-                for (Cookie cookie : cookies) {
-                    if (!cookie.name().equals(cookieName)) {
-                        newCookiesList.add(cookie);
+                if (!cookieName.equals("*")) {
+                    for (Cookie cookie : cookies) {
+                        if (!cookie.name().equals(cookieName)) {
+                            newCookiesList.add(cookie);
+                        }
                     }
                 }
                 cookieStore.put(domain, newCookiesList);
diff --git a/node_modules/react-native-ssl-pinning/ios/RNSslPinning/RNSslPinning.m b/node_modules/react-native-ssl-pinning/ios/RNSslPinning/RNSslPinning.m
index 53c4e54..4d55210 100644
--- a/node_modules/react-native-ssl-pinning/ios/RNSslPinning/RNSslPinning.m
+++ b/node_modules/react-native-ssl-pinning/ios/RNSslPinning/RNSslPinning.m
@@ -56,6 +56,8 @@ - (instancetype)init
 
         if([cookieName isEqualToString:name]) {
             [cookieStorage deleteCookie:cookie];
+        } else if([cookieName isEqualToString:@"*"]) {
+            [cookieStorage deleteCookie:cookie];
         }
     }
